csharp/src/Ice/Internal/DefaultsAndOverrides.cs

```diff
@@ -58,46 +58,6 @@ internal DefaultsAndOverrides(Ice.Properties properties, Ice.Logger logger)
             overrideTimeoutValue = -1;
```

Comment:
I think we should eliminate the "override timeout" in a follow-up PR (in C++ and C#).